nncf/experimental/common/tensor_statistics/collectors.py

```diff
@@ -304,22 +301,48 @@ def _aggregate(self) -> None:
             result[key] = val
         return result
 
-    def get_statistics(self) -> Union[TensorStatistic, Dict[str, Any]]:
+    def set_cache(self, statistics: TensorStatistic) -> None:
+        """
+        Sets cached statistics from given config and disable TensorCollector.
+        :param statistics: TensorStatistic.
+        """
+        self._cached_statistics = statistics
+        self.reset()
+        self.disable()
+
+    def create_statistics_container(self, config: Dict[str, Any]) -> TensorStatistic:
+        """
+        Returns a TensorStatistic instance with aggregated values.
+
+        :param config: Aggregated values.
+        :return: TensorStatistic instance.
+        """
+        if not self._stat_container:  # TODO: need to remove an ability to return a Dict.
```

Comment:
Please refactor to comply with the styleguide
https://github.com/openvinotoolkit/nncf/blob/develop/docs/styleguide/PyGuide.md#46-todo-comments